### ISSUE TITLE
WebAPI component does not send body or headers

### ISSUE THREAD

**Issue by lflfm** [thunkable/thunkable-issues#129]
### Platform
Thunkable X (using iOS)

### Issue/Request Origin
WebAPI component, tested using iPhone 8

### Steps to Reproduce
1. Create a new project
2. insert a button
3. insert a text box
4. insert WebAPI component
5. fill in information in the WebAPI component
6. Send a POST request to a server that just replies back the headers and body that it received
7. Write the server's response to the text box and see that the server received a blank header and blank body

### Expected Behavior
Headers and body should be sent

### Actual Behavior
Headers and body are not sent
```
![20190128_111107_01_screenshot](https://user-images.githubusercontent.com/26166377/51829468-dead3e80-22ed-11e9-8438-a8f9e9e35be5.png)
![20190128_111126_01_screenshot](https://user-images.githubusercontent.com/26166377/51829473-e0770200-22ed-11e9-9538-3367261ea942.png)

(sorry, no screenshot of actual results, but the server replied that header and body were empty)
```

**Comment by lflfm** [comment]
Hi Mark!
 I was creating a test REST server and I noticed that even the basic body element was missing in the node.js express request parameter - this seemed funny so I did little more digging and noticed that the Web API component does not send the Content-Type header at all (which actually makes sense that it isn't set by default, although something needs to be set if there's a body, right...).
  By setting it manually, it worked; so we can close this bug - I have opened a feature request to set headers via block code (issue 131).
  I did, however, put in a pull request for a change in the Web API component documentation so that this is mentioned; pull request #19 if you care to have a look (https://github.com/thunkable/docs-thunkable-com/pull/19).
